config/boot.rb

```diff
@@ -3,9 +3,3 @@
 require "bundler/setup" # Set up gems listed in the Gemfile.
 require "bootsnap/setup" # Speed up boot time by caching expensive operations.
 
```

Comment:
where does dotenv get loaded now?

Reply:
the environment is loaded using docker-compose the .env.local